_posts/2021-09-28-str-format.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "String Formatting"
+date: 2021-09-28
+author: Arjun
+category: Tutorial
+---
+
+<h2>Here is the list of complete set of symbols which can be used along with "%"</h2>
+
+**%c** character
+```python
+>>> print ("%c" %'h')
+h
+>>> print ("%c" %104)
+h
+>>> print ("%c" %'"')
+"
+>>> print ("%c" %34)
+"
+```
+
+**%s**	string conversion via str() prior to formatting
+```python
+>>> x = 'python'
+>>> print("Hello %s , %s to SharetoLearn!" %(x,"Welcome"))
+Hello python , Welcome to SharetoLearn!
+```
+
+**%i**	signed decimal integer
+```python
+>>> print("Hello %i + %i = %i" %(5,5,10))
+Hello 5 + 5 = 10
+```
+
+**%d**	signed decimal integer
+```python
+>>> print("%d" % 100)
+100
+>>> print("%d" % 0b1111)
+15
+>>> print("%03d" % 1)
+001
+>>> print("%03d" % 100.111)
+100
+>>> print("%03d" % 10.111)
+010
+```
+
+**%u**	unsigned decimal integer
+```python
+>>> u'Hello\u0020Python ! '
+u'Hello Python ! '
+```
```

Comment:
Give another example where you get a character from Indian language. Or may be the Rupee symbol. It's more fun :-)